e2e_tests/integration/viz.spec.ts

```diff
@@ -44,4 +44,32 @@ describe('Viz rendering', () => {
     )
     cy.executeCommand('MATCH (a:TestLabel) DETACH DELETE a')
   })
+  it('can change default color of nodes', () => {
+    const selectorNodeLabelAll =
+      '[data-testid="property-details-overview-node-label-*"]'
```

Comment:
There are some `data-testid`s written with camel case, while some are kebab case, in this project. I think we need to unify all of them, or even using variables to set these ids, in the future.

Reply:
Probably we can clean them up in your subsequent PR.